Tools/OSD/osd_screen.cs

```diff
@@ -604,12 +606,6 @@ private void rbtSortAlphabetic_CheckedChanged(object sender, EventArgs e)
 			}
 		}
 		
-        private int correct_NTSC(int y){
-            if (y >= osd.getCenter() && osd.is_ntsc()) 
-                y+=3;
-            return y;
-        }
-
```

Comment:
what has prevented this feature? Occam's Razor - it is not necessary to produce the spirit, it is not necessary to embed checking code in every place! It must to split out to a function